src/command/mcl.zig

```diff
@@ -1896,6 +2011,227 @@ fn mclWaitRecoverSlider(params: [][]const u8) !void {
     }
 }
 
+/// Add logging configuration for registers logging in the specified line
+fn addLogRegisters(params: [][]const u8) !void {
+    const line_name = params[0];
+    const line_idx = try matchLine(line_names, line_name);
+    const line = mcl.lines[line_idx];
+
+    var log: LogLine = undefined;
+
+    // Validate "axes" parameter
+    var axis_input_iterator = std.mem.tokenizeSequence(u8, params[1], ",");
+    while (axis_input_iterator.next()) |token| {
+        const axis_index = try std.fmt.parseInt(mcl.Station.Id, token, 0) - 1;
```

Comment:
This is not the correct type to use for an axis, I'd suggest mcl.Axis.Id.Line

Additionally, subtracting before validation may cause an underflow here, which would break runtime safety checks.